### PR TITLE
Formatted options as post-encode arguments (to master)

### PR DESCRIPTION
**Pull request by milanjaros**
Added possibility to use formatted arguments as post-encode arguments. 
There are two commits: first one with business logic and second one with few UI tweaks (not important).

### CODE REVIEW

File: VidCoder/Resources/PickerRes.resx [hunk 1]
```diff
@@ -250,4 +250,7 @@
     <value>Forced only</value>
     <comment>Label for checkbox in Picker window to choose only Forced subtitles of the chosen subtitle track.</comment>
   </data>
+  <data name="OverrideNameFormatLabel" xml:space="preserve">
+    <value>Available arguments: {source} {title} {range} {preset} {date} {time} {quality} {parent} {titleduration}</value>
```
Comment (RandomEngy) [inline VidCoder/Resources/PickerRes.resx]: I think this value should be Available arguments: {0} and have the {source}, etc hard-coded in the code, as that part should not get translated.

**Comment by RandomEngy** [comment]
Thanks for the pull request! Can you retarget to the beta branch? That way I can merge it sooner, make sure there's no issues and get translations for the new strings.

File: VidCoder/Resources/PickerRes.resx (hunk 1 above)
Comment (milanjaros) [reply VidCoder/Resources/PickerRes.resx]: Thanks for feedback. Hmm. It depends. I know the {placeholders} are part of code and you could possibly change/extend it "one day". On the other hand this solution gives you possibility to explain some particular {placeholder}, e.g. {range} and {parent} are not obvious from my point of view.

**Comment by RandomEngy** [comment]
It looks like something went wrong with the retargeting to beta; it shows 242 files changed. Can you rebase or open another PR with some cleaner changes?

File: VidCoder/Resources/PickerRes.resx (hunk 1 above)
Comment (RandomEngy) [reply VidCoder/Resources/PickerRes.resx]: I'd rather link to the webpage that explains all of them if I want to explain them. The purpose of this string is simply to list all of the available ones tokens.

**Pull request closed without merging**